index.html

```diff
@@ -3304,19 +3278,20 @@ <h3><dfn>Writing content</dfn></h3>
             <ol>
               <li>
                 If |key| equals "`signal`" and |value| is not `undefined`, set
-                |reader|.<a>[[\Signal]]</a> to |value|.
+                |reader|.<a>[[\ScanOptions]]</a>'s |signal:AbortSignal| to |value|.
```

Comment:
We are getting rid of scan options so why not just [[\ScanSignal]] to |value|

Reply:
To be symmetric with write, and to be future-proof if we will eventually add other scan options later.